examples/openthread/hello/Makefile

```diff
@@ -0,0 +1,17 @@
+# Makefile for user application
+
+# Specify this directory relative to the current application.
+TOCK_USERLAND_BASE_DIR = ../../..
+
+# Specify this app depends on the MTD OpenThread library.
+include $(TOCK_USERLAND_BASE_DIR)/libopenthread/libopenthread-mtd.mk
+
+# Arbitrarily set this for now. We will need to revisit these values.
+#APP_HEAP_SIZE:=5000
+STACK_SIZE:=5000
```

Comment:
What should these be?

Reply:
I picked a larger than the default value for testing somewhat arbitrarily. For our version we upstream, I'm not sure the best way determine what these should be. @bradjc do you have thoughts on a good way to determine what this value should be?